openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-settings.js

```diff
@@ -357,9 +357,11 @@ var VideoSettings = (function() {
 	}
 	function _getDevConstraints(callback) {
 		const devCnts = {audio: false, video: false};
-		if (navigator.mediaDevices === undefined) {
-			OmUtil.error('Browser multimedia devices are blocked by security settings. Try https.');
-			callback(devCnts);
+		if (window.isSecureContext === false) {
+			OmUtil.error($('#settings-https-required').text());
+		}
```

Comment:
I believe `return` should be added here

Reply:
Inside first `if`

Reply:
I was copying it from
https://github.com/apache/openmeetings/blob/master/openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-settings.js#L377

The effect was that both errors are displayed. 

But I added a return now. And it only shows the localised one.